### PR TITLE
remove kafka volume when clean up

### PR DESCRIPTION
**Pull request by xin-cai**
remove kafka volume when clean up

### CODE REVIEW

**Comment by xin-cai** [comment]
PG4/290 is running

**Comment by xin-cai** [comment]
PG4/290 approved

**Comment by sven-lange-last** [comment]
Observations regarding Docker volumes from Kafka container:

- The `Dockerfile` for Kafka creates two unnamed volumes: https://hub.docker.com/r/ches/kafka/~/dockerfile/
- `docker rm` won't remove volumes unless option `-v` is specified: https://docs.docker.com/v1.12/engine/reference/commandline/rm/
- The `clean`script of Ansible `kafka` role only asserts that the `kafka` Docker container is absent: https://github.com/apache/incubator-openwhisk/blob/master/ansible/roles/kafka/tasks/clean.yml
- The Ansible `docker_container` module support volume management via parameter `keep_volumes` which is set to `true` by default: https://docs.ansible.com/ansible/docker_container_module.html. 

In other words: at least for the `kafka` container, we don't clean up volumes when removing containers but retain them such that disk space fills up if volumes are not removed otherwise.

This PR fixes said problem. Looks good to me.

**Comment by xin-cai** [comment]
The travis build ran exceed the maximum allowed time, unfortunately I don't have the right to restart the build... @jeremiaswerner could you help me to re-trigger the travis build? Thanks! 
https://travis-ci.org/apache/incubator-openwhisk/builds/235197294?utm_source=github_status&utm_medium=notification

**Comment by xin-cai** [comment]
PG4/299 approved

**Comment by xin-cai** [comment]
@jeremiaswerner  could you pls mergen? I don't have right to assign either reviewer or assignees ...
